career4.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>TechHub-Careers-data-security-analyst</title>
+    </head>
+    <body>
+        <header>
+            <div class="nav-container">
+                <div id="nav-branding">
+                    <h1>Tech<span>Hub</span>-Careers</h1>
+                </div>
+                <nav>
+                    <ul>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="#">Careers</a></li>
+                        <li><a href="#">Wait-List</a></li>
```

Comment:
Waitlist should be one word
```suggestion
                        <li><a href="#">Waitlist</a></li>
```